packages/block-library/src/image/edit.js

```diff
@@ -286,6 +287,11 @@ export function ImageEdit( {
 		className: classes,
 	} );
 
+	const { showDuotoneControls } = useDispatch( blockEditorStore );
+	useEffect( () => {
+		showDuotoneControls( clientId, !! url );
+	}, [ url, clientId ] );
```

Comment:
Since values in the `blockEditor` store, values don't persist. I wanted those values to be restored on refresh. I also want to start with the duotone toolbar disabled in the image block but start enabled for the blocks that just add support in block.json.

In order to do those things, I ended up using this `useEffect` hook to enable it when the url was set. I'm not a huge fan of the hook, so I'm investigating other approaches.